packages-exp/functions-exp/src/callable.test.ts

```diff
@@ -98,7 +101,7 @@ describe('Firebase Functions > Call', () => {
 
   it('scalars', async () => {
     const functions = createTestService(app, region);
-    const func = httpsCallable(functions, 'scalarTest');
+    const func = httpsCallable<number, number>(functions, 'scalarTest');
```

Comment:
I believe at least the request (and maybe the response as well) *must* be objects. Scalars may not be allowed by the server-side implementation.

Reply:
This is a live test that sends a number to the server and successfully gets a number response so it seems to work, unless I'm misunderstanding. To make sure, I also created a new `onCall` function that logged a number sent to it and returned a number and tried it from a test app and it seems to work:
```
exports.numberTest = functions.https.onCall((data) => {
    console.log(data, typeof data);
    return(44);
});
```
Let me know if I misunderstood.